### PR TITLE
added Hyper project to the projectlist

### PR DESCRIPTION
**Pull request by nagasaisriya**
Added Hyper to the project list #173

### CODE REVIEW

File: src/components/ProjectList/listOfProjects.js [hunk 1]
```diff
@@ -196,14 +196,19 @@ const projectList = [
...
     imageSrc: 'https://avatars0.githubusercontent.com/u/9892522?v=3&s=100',
     projectLink: 'https://github.com/freeCodeCamp/mail-for-good/contribute',
     description: 'An open source email campaign management tool.',
     tags: ['Nodejs', 'JavaScript', 'Email-Campaigns'],
-  },
-  {
+  },{
+    name: 'Hyper',
+    imageSrc: '68747470733a2f2f6173736574732e76657263656c2e636f6d2f696d6167652f75706c6f61642f76313534393732333834362f7265706f7369746f726965732f68797065722f68797065722d332d7265706f2d62616e6e65722e706e67',
+    projectLink: 'https://github.com/vercel/hyper',
```
Comment (sara-02) [inline src/components/ProjectList/listOfProjects.js:208]: Better point to https://github.com/vercel/hyper/contribute

File: src/components/ProjectList/listOfProjects.js [hunk 2]
```diff
@@ -196,14 +196,19 @@ const projectList = [
...
     name: 'Mail For Good',
     imageSrc: 'https://avatars0.githubusercontent.com/u/9892522?v=3&s=100',
     projectLink: 'https://github.com/freeCodeCamp/mail-for-good/contribute',
     description: 'An open source email campaign management tool.',
     tags: ['Nodejs', 'JavaScript', 'Email-Campaigns'],
-  },
-  {
+  },{
+    name: 'Hyper',
+    imageSrc: '68747470733a2f2f6173736574732e76657263656c2e636f6d2f696d6167652f75706c6f61642f76313534393732333834362f7265706f7369746f726965732f68797065722f68797065722d332d7265706f2d62616e6e65722e706e67',
```
Comment (sara-02) [inline src/components/ProjectList/listOfProjects.js:207]: Broken image link, please fix.

**Review by sara-02 — CHANGES_REQUESTED** [review]

File: src/components/ProjectList/listOfProjects.js (hunk 1 above)
Comment (nagasaisriya) [reply src/components/ProjectList/listOfProjects.js:208]: Thank you for reviewing! I'll try to resolve it.

File: src/components/ProjectList/listOfProjects.js (hunk 2 above)
Comment (nagasaisriya) [reply src/components/ProjectList/listOfProjects.js]: Hello @sara-02  gentle reminder, I've updated the changes, can you please review and merge the pr.
Thanks in advance

**Review by sara-02 — APPROVED** [review]

**Pull request merged**